### DISCUSSION TITLE
Sveltekit style +route.tsx file

### DISCUSSION THREAD

**Discussion by PuruVJ** [remix-run/remix#8763] [Q&A]
Is there a way I can tell remix to treat the file `+route.jsx` as the file for its route? plus symbol hoists the file to the top in VSCode's file view, making it easy to access the route file

**Comment by alexanderson1993** [comment]
Yep, you absolutely can do that. You can use the `routes` config to define custom routes - they don't even have to be part of the filesystem. Add a bit of dynamic filesystem crawling and matching, and you can create any style of filesystem routes that you want! https://remix.run/docs/en/main/file-conventions/remix-config#routes

Here's an example repo that does just that: https://github.com/kiliman/remix-flat-routes
